src/datasources/targeted-messaging/entities/outreach.entity.ts

```diff
@@ -3,6 +3,11 @@ export class Outreach {
   name: string;
   start_date: string;
   end_date: string;
+  source_id: number;
+  type: string;
+  team_name: string;
+  source_file: string;
+  source_file_processed_date: string;
```

Comment:
Both `source_file` and `source_file_processed_date` can be `null`. I'm not 100% sure, but as it's a `TIMESTAMP`, shouldn't `source_file_processed_date` be a `Date` if defined instead of a `string`?

Reply:
I initially thought we would need to thread these values as strings when dealing directly with the `postgres` library, but you are right, it seems we can use `Date` directly so need to parse them twice. I've changed it in [bfb327e](https://github.com/safe-global/safe-client-gateway/pull/2010/commits/bfb327e54f8e3c1713ed4922a6f61d1ed48151c2)

This can also be applied to the other DB-related entities like `DbTargetedSafe` or `DbSubmission`. Will change those in a separate PR as they are not related to this one.

Reply:
We should definitely check/adjust for the `Date` coercion accordingly when we move this to the ORM as that uses `pg` instead of `postgres`, cc @PooyaRaki